backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/shippingNotification/shippingNotification.feature

```diff
@@ -0,0 +1,195 @@
+      | ps_1       | pricing_system_name_21092023 | pricing_system_value_21092023 | pricing_system_description_21092023 | true         |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name                       | OPT.Description | SOTrx | IsTaxIncluded | PricePrecision | OPT.IsActive |
+      | pl_1       | ps_1                          | DE                        | EUR                 | price_list_name_15112022_2 | null            | true  | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name                    | ValidFrom  |
+      | plv_1      | pl_1                      | salesOrder-PLV_21092023 | 2021-04-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_1       | plv_1                             | p_1                     | 10.0     | PCE               | Normal                        |
+      | pp_2       | plv_1                             | p_2                     | 10.0     | PCE               | Normal                        |
+    And metasfresh contains C_BPartners without locations:
+      | Identifier    | Name                 | M_PricingSystem_ID.Identifier | OPT.IsCustomer |
+      | endcustomer_1 | Endcustomer_21092023 | ps_1                          | Y              |
+    And metasfresh contains C_BPartner_Locations:
+      | Identifier        | GLN           | C_BPartner_ID.Identifier | OPT.IsBillToDefault |
+      | bpLocationDefault | 0123456789012 | endcustomer_1            | Y                   |
+    And metasfresh contains AD_Users:
+      | AD_User_ID.Identifier | Name                      | OPT.C_BPartner_ID.Identifier | OPT.C_BPartner_Location_ID.Identifier | OPT.IsBillToContact_Default |
+      | bpContactDefault      | bpContactDefault_21092023 | endcustomer_1                | bpLocationDefault                     | Y                           |
+    And metasfresh contains C_Auction:
+      | Identifier | Name     | Date       |
+      | Auction_1  | Auction1 | 2023-09-21 |
+
+
+  @from:cucumber
+  Scenario: we can generate shipping notifications for sales order and data is passed correctly to the notification and lines
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID.Identifier | OPT.C_BPartner_Location_ID.Identifier | OPT.AD_User_ID.Identifier | DateOrdered | OPT.AD_Org_ID.Identifier | OPT.POReference      | OPT.C_Harvesting_Calendar_ID.Identifier | OPT.Harvesting_Year_ID.Identifier | OPT.Description            | OPT.C_Auction_ID.Identifier | OPT.DeliveryRule | OPT.DeliveryViaRule | OPT.M_Warehouse_ID.Identifier | OPT.M_Locator_ID.Identifier |
```

Comment:
Pls take out: OPT.DeliveryRule | OPT.DeliveryViaRule
They are not needed for this test.